src/alire/alire-solutions-diffs.adb

```diff
@@ -384,7 +384,10 @@ package body Alire.Solutions.Diffs is
       if Changed then
          Table.Print (Level);
       else
-         Trace.Log (Prefix & "No changes between former an new solution.",
+         Trace.Warning (Boolean'(This.Former  = This.Latter)'Image);
```

Comment:
`'Image` should be `'Img` here to avoid "prefix of "Image" attribute must be a type" error.

Reply:
Strange. What compiler version is this?

Reply:
GNAT FSF 7.5.

Reply:
I realize now that those were forgotten logs to be removed, so not a real issue.